guides/source/active_storage_overview.md

```diff
@@ -82,6 +82,14 @@ To use the Amazon S3 service in production, you add the following to
 config.active_storage.service = :amazon
 ```
 
+To use the test service when testing, you add the following to
+`config/environments/test.rb`:
+
+```ruby
+# Store files locally.
```

Comment:
What do you think to add an explanation that the files are stored in a temporary directory? 
I think that it is better to clarify the difference with `development`. 

For example, in `test.rb` generated by default, the following explanation is made.
https://github.com/rails/rails/blob/9422433e1a875a3f3c3179fff32fb05858604dc6/railties/lib/rails/generators/rails/app/templates/config/environments/test.rb.tt#L33

Reply:
Good call @y-yagi, just updated the comment so they're the same (and force-pushed to keep the history clean)